### PR TITLE
Fixed some of review comments

### PR DESCRIPTION
**Pull request by anpolimus**
**Related Issue/Ticket:**

@hamrant I was not fixing plugin methods, since you were implementing pluginable structure and could fix it better.

(Replace this line with a link to the related GitHub issue on [ymcatwincities/openy_gated_content](https://github.com/ymcatwincities/openy_gated_content/issues) or your own ticketing system)

## Steps to test:

- [ ] First do this
- [ ] Then do this (and look at this great screenshot).
- [ ] Finally observe the issue is resolved.

## Quality checks:

Please check these boxes to confirm this PR covers the following cases:

- Maintaining our upgrade path is essential. Check one or the other:
  - [ ] This PR  provides updates via `hook_update_N` or other means.
  - [ ] No updates are necessary for this change.
- Front end fixes should be tested against all of the Open Y Themes.
  - [ ] Tested against Carnation
  - [ ] Tested against Lily
  - [ ] Tested against Rose
  - [ ] This change does not contain front-end fixes.
- [ ] I have flagged this PR "Needs Review" or pinged the VY devs/QA
  team in Slack

### CODE REVIEW

**Review by hamrant — APPROVED** [review]

File: openy_gated_content.api.php [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+ *
+ * Every module could alter initial Virtual Y Vue JS app settings before
+ * sending it to the front.
+ *
+ * @param array $backend_info
+ *   Virtual Y app settings.
+ *
+ */
+function hook_virtual_y_app_settings_alter(array &$backend_info) {
+  $config['personal_training_enabled'] = TRUE;
```
Comment (podarok) [inline openy_gated_content.api.php:25]: where $config comes from?

**Review by podarok — CHANGES_REQUESTED** [review]
small one

File: openy_gated_content.api.php (hunk 1 above)
Comment (anpolimus) [reply openy_gated_content.api.php]: fixed

**Review by podarok — APPROVED** [review]

**Comment by anpolimus** [comment]
Tested code. Nothing was broken.
Merging.

**Pull request merged**